editor/src/editor/layout/console.tsx

```diff
@@ -8,6 +8,7 @@ import { Editor } from "../main";
 import { ContextMenu, ContextMenuContent, ContextMenuItem, ContextMenuTrigger } from "../../ui/shadcn/ui/context-menu";
 
 import { EditorConsoleProgressLogComponent } from "./console/progress-log";
+import { UniqueNumber } from "../../export";
```

Comment:
Ah! You have the same auto-complete I have on VSCode, can you replace by `import { UniqueNumber } from "../../tools/tools";`